### PR TITLE
doc(system-backup/restore): add feature doc

### PR DESCRIPTION
**Pull request by c3y1huang**
https://github.com/longhorn/longhorn/issues/1455

### CODE REVIEW

**Comment by c3y1huang** [comment]
TODO:
- https://github.com/longhorn/longhorn-manager/pull/1489#discussion_r1041905414
- https://github.com/longhorn/longhorn-manager/pull/1489#discussion_r1041906826

File: content/docs/1.4.0/advanced-resources/system-backup-restore/restore-longhorn-system.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+## Restart Longhorn System Restore
+
+1. [Delete Longhorn System Restore](#delete-longhorn-system-restore) that is in progress.
+1. [Create Longhorn System Restore](#create-longhorn-system-restore).
+
+## Configurable Settings
+
+Some settings are excluded as configurable before the Longhorn system restore.
+- Concurrent volume backup restore per node limit
```
Comment (c3y1huang) [inline content/docs/1.4.0/advanced-resources/system-backup-restore/restore-longhorn-system.md:113]: TODO: add reference link after https://github.com/longhorn/website/pull/625 merged.

**Review by innobead — DISMISSED** [review]
LGTM. 

nit: We should mention the volume will not be triggered when doing system backup, so need to ensure the original volume having backup for the following system restore.

File: content/docs/1.4.0/terminology.md [hunk 2]
```diff
@@ -196,5 +196,15 @@ These files will still be available after a container crashes, but they will not
 
 For more information, see the Kubernetes documentation on [volumes.](https://kubernetes.io/docs/concepts/storage/volumes/)
 
+### Volume Backup
```
[the remark this answers is not in the document]
Comment (c3y1huang) [reply content/docs/1.4.0/terminology.md]: Updated.

**Comment by c3y1huang** [comment]
> nit: We should mention the volume will not be triggered when doing system backup, so need to ensure the original volume having backup for the following system restore.

Updated.

**Review by innobead — APPROVED** [review]

**Pull request merged**